core/arch/arm32/kernel/tee_ta_manager.c

```diff
@@ -625,6 +625,9 @@ static TEE_Result tee_ta_load(const kta_signed_header_t *signed_ta,
 #if defined(CFG_SE_API)
 	ctx->se_service = NULL;
 #endif
+#if !defined(CFG_TEE_CORE_NO_TA_TRACE)
+	ctx->trace = true;
+#endif
```

Comment:
Can ctx->trace be set to false somewhere else? If not, this is a global build-time setting which does not need to be stored in the context. Or did I miss something?